frontend/packages/policy-editor/src/components/PolicyCardRules/PolicyRule/PolicyAccessPackages/PolicyAccordion/PolicyAccordion.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useId, useState } from 'react';
+import cn from 'classnames';
+import { StudioButton, StudioLabelAsParagraph } from '@studio/components';
+import * as StudioIcons from '@studio/icons';
+import { ChevronDownIcon, ChevronUpIcon } from '@studio/icons';
+import classes from './PolicyAccordion.module.css';
+
+interface PolicyAccordion {
+  icon?: string;
+  title: string;
+  subTitle: string;
+  extraHeaderContent?: React.ReactNode;
```

Comment:
Same as commented earlier 😄 
```suggestion
  extraHeaderContent?: ReactNode;
```